packages/storybook/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@nrwl/storybook",
+  "version": "0.0.1",
+  "description": "Storybook plugin for Nx",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nrwl/nx.git"
+  },
+  "keywords": [
+    "Monorepo",
+    "Angular",
+    "React",
+    "Web",
+    "Node",
```

Comment:
I don't know if this one applies. Sure it runs in Node but such is probably true for everything in NPM lol...

Reply:
I'm not familiar with what should be added as `keywords` but this may help: 

in a default angular project using `sb init` to bootstrap Storybook the `.storybook/tsconfig.json` has `types: ["node"]` https://github.com/storybookjs/storybook/blob/69d9c98d3be0c845c0b8bab96a099c4392364a78/lib/cli/generators/ANGULAR/template/.storybook/tsconfig.json

I added it myself but don't right away know why I added it - if this is of interest I can look it up for you, just let me know. So yes, kinda everything is "node" in npm but Storybook explicitly needs it - may be a reason to add it here?